tests/scripts/mypy.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -e
+
+mypy --install-types --non-interactive .
```

Comment:
NIT: if this script is only calling `mypy` what do you think to remove it and add the call in the above consumers?

Reply:
same as  above